index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My-portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com" crossorigin="">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=poppins:wght@600&display=swap">
+
+</head>
+<body>
+    <header class="header">
+		<section class="nav">
+			<nav class="logo">
+			   <a href=""><h1>My Logo</h1></a>
+			</nav>
+			<nav class="burger">
+                <ul>
+                    <li>
+                        <label id="checkbox_toggle" class="burger">&#9776;</label>
+                    </li>
+                 </ul>
+        
+			</nav>
+		</section>
+	</header>
+	<main class="main">
+		<section class="top">
+				<div class="top-item">
+					<div class="intro">
+						<h1>I'm Nickson <br> Glad to see you!</h1>
+					</div>
+					<div class="about">
+						<p>I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+					</div>
+					<div class="description">
+						<h2 class="connect">LET'S CONNECT</h2>
+                    </div>
+                    
+                    <ul>
+                        <li class="icons">
+                            <img src="images/social_media_icons.png" alt="">
+                        </li>
```

Comment:
- [ ] Here, please don't use one image. Instead, create 5 list elements with links(use anchor tag). The reason behind this, this section is where you are going to link your accounts to the icons in the upcoming tasks.